index.js

```diff
@@ -114,6 +114,16 @@ function SendStream(req, path, options) {
     ? normalizeList(opts.extensions, 'extensions option')
     : []
 
+  if (Array.isArray(opts.precompressed)) {
+    this._precompressionFormats = opts.precompressed
+  } else if (opts.precompressed) {
+    this._precompressionFormats = [{encoding: 'br', extension:'.br'}, {encoding:'gzip', extension:'.gz'}]
```

Comment:
The comment I linked to has link to draft specification (https://github.com/madler/brotli/blob/master/br-format-v3.txt) for the .br file format. It gives an example of .tar.br as a file name.